hedera-dependency-versions/build.gradle.kts

```diff
@@ -51,19 +52,18 @@ dependencies.constraints {
         api(gav("com.google.jimfs", "1.2"))
         api(gav("com.google.protobuf", protobufVersion))
         api(gav("com.google.protobuf.util", protobufVersion))
-        api(gav("com.hedera.hashgraph.protobuf.java.api", "0.40.0-blocks-state-SNAPSHOT")) // TODO removed through other PR
         api(gav("com.hedera.pbj.runtime", "0.7.1"))
         api(gav("com.sun.jna", "5.12.1"))
         api(gav("com.swirlds.base", swirldsVersion))
         api(gav("com.swirlds.cli", swirldsVersion))
         api(gav("com.swirlds.common", swirldsVersion))
-        api(gav("com.swirlds.config", swirldsVersion))
+        api(gav("com.swirlds.config.api", swirldsVersion))
```

Comment:
Renamed `com.swirlds.config` -> `com.swirlds.config.api` to be consistent with the Gradle subproject (folder) name.

Reply:
I wasn't aware of these discussions.

Alternatively to what I did here, we can keep the Module Name and instead rename the gradle subproject. I can do the adjustment in this PR, if the names without "api" is what was decided for.

Should I do that change?